src/aux-common/bots/BotEvents.ts

```diff
@@ -144,6 +144,19 @@ export type AsyncActions =
     | RpioSPITransferAction
     | RpioSPIWriteAction
     | RpioSPIEndAction
+    | SerialConnectAction
+    | SerialStreamAction
+    | SerialOpenAction
+    | SerialUpdateAction
+    | SerialWriteAction
+    | SerialReadAction
+    | SerialCloseAction
+    // | SerialSetAction
```

Comment:
@WesleyMayle Can you remove this code if it is no longer needed?

Reply:
Sure, I can remove it for now.